modules/specialized-guides/pages/salt/salt-formula-saltboot.adoc

```diff
@@ -142,4 +142,4 @@ On the ``msdos`` disk label, you can create a maximum of four primary partition
 Extended partitions are not supported.
 If you need more than four partitions, use the ``GPT`` disk label instead.
 
-For more information on troubleshooting problems with the {saltboot} formula, see xref:administration:troubleshooting/tshoot-saltboot.adoc[].
+// For more information on troubleshooting problems with the {saltboot} formula, see xref:administration:troubleshooting/tshoot-saltboot.adoc[].
```

Comment:
I'd completely remove this sentence with the xref and the empty line before.

Reply:
Done.